.github/workflows/release-tests.yml

```diff
@@ -0,0 +1,32 @@
+name: Release tests
+
+on: workflow_dispatch
+
+env:
+  DEFAULT_PYTHON: 3.8
+
+jobs:
+  virtualenv-15-windows-test:
+    name: Regression test for virtualenv==15.1.0 on Windows
```

Comment:
Last two 🙂

I believe it's possible to add comments in workflow files too. It would be great if we could link to the PR here just to have a reference later why it was created.